core-field-editors/package.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Not a big fan of having this extra overhead of keeping the shrinkwrap in sync and clutter free.